package.json

```diff
@@ -50,10 +50,10 @@
     "build": "grunt",
     "test": "grunt test",
     "test-fast": "grunt test-fast",
-    "version": "node build/sri-update",
-    "prepublishOnly": "grunt build && node build/sri-update --validate",
+    "version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
+    "prepublishOnly": "grunt build",
     "postinstall": "node build/utils/postinstall.js",
-    "release": "standard-version"
+    "release": "standard-version && node build/sri-update"
```

Comment:
@marcysutton Are you alright with removing this, now that Attest-master is set up to publish axe-core?

Reply:
Oh yes, I'd love to get a demo from you on using that tool!